frontend/simple/main.js

```diff
@@ -45,6 +47,14 @@ var router = new Router({
       name: CreateGroup.name,
       meta: {
         title: 'Create Group'
+      },
+      beforeEnter (to, from, next) {
+        if (!store.state.loggedIn) {
+          console.log(to.name, `redirecting to ${SignUp.name}!`)
+          next({ path: '/signup' })
```

Comment:
How is `/signup` then going to redirect back to Create Group? This was the functionality that was deleted and that I still need to verify is fixed per #127.